sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clientTelemetry/ClientTelemetry.java

```diff
@@ -217,17 +217,18 @@ private void readHistogram() {
         }
     }
 
-    private void fillMetricsInfo(ReportPayload payload, DoubleHistogram histogram) {
-        payload.getMetricInfo().setCount(histogram.getTotalCount());
-        payload.getMetricInfo().setMax(histogram.getMaxValue());
-        payload.getMetricInfo().setMin(histogram.getMinValue());
-        payload.getMetricInfo().setMean(histogram.getMean());
+    private void fillMetricsInfo(ReportPayload payload, ConcurrentDoubleHistogram histogram) {
+        DoubleHistogram copyHistogram = histogram.copy();
```

Comment:
does this have a perf implication? please do a quick perf test to see if there is any impact.

Reply:
No perf implication , this is just reference which i change from DoubleHistogram  to ConcurrentDoubleHistogram , object was already ConcurrentDoubleHistogram  in the code.

Reply:
Updating the comment , as my above reply was for different comment.
New reply - "No perf impact, this is separate 10 min scheduled thread, wont interfere with current workload"